### DISCUSSION TITLE
Win+A opens only action center, not also notifications

### DISCUSSION THREAD

**Discussion by BlackRule** [valinet/ExplorerPatcher#106] [Q&A] [answered]
And if I disable control center win+A doesn't do anything.
How to make win+A  open notifications as in Win10 ?

**Comment by valinet** [comment, marked as answer]
`Win`+`N` is now the hotkey used for notifications in Windows 11. They configured `Win`+`A` exclusively for Control Center. If you disable that, indeed, it remains unused. Idk what to say:
* a quick Autohotkey script...? 
* should this be implemented here...? 
* just use `Win`+`N` instead...

**Reply by BlackRule** [reply]
Autohotkey script works
#a::
Send #n
return
Thanks

**Reply by valinet** [reply]
You're welcome. Thanks for sharing back.
